ui/raidboss/data/05-shb/trial/emerald_weapon-ex.txt

```diff
@@ -61,6 +60,9 @@ hideall "--sync--"
 1031.3 "Primus Terminus Est" sync /:The Emerald Weapon:55C3:/
 1039.2 "--sync--" sync /:The Emerald Weapon:5B1C:/
 1046.4 "Tertius Terminus Est" sync /:The Emerald Weapon:55CC:/
+1048.4 "Tertius Terminus Est" sync /:Bitblade:55CD:/
+1049.6 "Tertius Terminus Est" sync /:Bitblade:55CD:/
+1050.8 "Tertius Terminus Est" sync /:Bitblade:55CD:/
```

Comment:
```suggestion
1048.4 "Tertius Terminus Est" # sync /:Bitblade:55CD:/
1049.6 "Tertius Terminus Est" # sync /:Bitblade:55CD:/
1050.8 "Tertius Terminus Est" # sync /:Bitblade:55CD:/
```

Here and elsewhere, we *definitely* don't want to sync when they're this close together!